api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/BaseDaoImpl.java

```diff
@@ -440,8 +440,8 @@ protected boolean containsAlias(Iterator<CriteriaImpl.Subcriteria> subcriteriaIt
 			int eIdx = plainString.indexOf('e');
 			if (dotIdx == -1 && eIdx == -1) {
 				return Optional
-				        .of(between(propertyName, quantityParam.getValue().subtract(new BigDecimal(0.1)).doubleValue(),
-				            quantityParam.getValue().add(new BigDecimal(0.1)).doubleValue()));
+				        .of(between(propertyName, quantityParam.getValue().subtract(new BigDecimal(0.5)).doubleValue(),
+				            quantityParam.getValue().add(new BigDecimal(0.5)).doubleValue()));
 			} else if (eIdx != -1) {
```

Comment:
@ibacher We'll need to correct this part

Reply:
So, `new BigDecimal(0.1)` is closer to what we want. What we actually want is this:

```java
// At the top of the file
private static final BigDecimal APPROX_RANGE = new BigDecimal(0.1);

BigDecimal value = quantityParam.getValue();
BigDecimal approxRange = APPROX_RANGE.multiply(value);

return Optional
    .of(between(propertyName, value.subtract(approxRange).doubleValue(), value.add(approxRange).doubleValue()));